src/templates/email/confirm_login_text.txt

```diff
@@ -1,9 +1,13 @@
 Dear {{ username }},
 
-To confirm your login, click on the following link:
+If you have recently tried to login to your CTFOJ account, click on the following link to confirm your login:
 
 {{ url_for('confirm_login', token=token, _external=True) }}
 
-If you have not tried to login in, please change your password as it may have been compromised.
+Otherwise, your password has been compromised and you should change it immediately at the following link:
 
-{{ CLUB_NAME }}
+{{ url_for('changepassword', _external=True) }}
```

Comment:
shouldn't there be a token here?

Reply:
nope, changepassword doesn't take a token. this redirects them to the login page, and then to the changepassword page.

Reply:
Ohh, right. I forgot. I was thinking of the forgotpassword page. Looks good so far, then.